README.rst

```diff
@@ -1029,6 +1029,58 @@ You can also change the parameters used for iperf checks (e.g. timing, port, use
 password, rsa_publc_key etc) using the `OPENWISP_MONITORING_IPERF_CHECK_CONFIG
 <#OPENWISP_MONITORING_IPERF_CHECK_CONFIG>`_ setting.
 
+Iperf Check Parameters
+######################
+
+Currently, iperf check supports the following parameters:
```

Comment:
I would move this to the end of the "Iperf" section. I would add links to this in the following sections of the documentation: 

1. ["3. Set up and configure Iperf server settings"](https://github.com/openwisp/openwisp-monitoring/blob/0ae7702d3cd0ddea77c7a075046697f644f9ba07/README.rst#3-set-up-and-configure-iperf-server-settings)
2. ["OPENWISP_MONITORING_IPERF_CHECK_CONFIG"](https://github.com/openwisp/openwisp-monitoring/blob/0ae7702d3cd0ddea77c7a075046697f644f9ba07/README.rst#openwisp_monitoring_iperf_check_config)

Reply:
I'll update that :+1:  thanks!